.gitignore

```diff
@@ -58,3 +58,6 @@ build-iPhoneSimulator/
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+# Sample bug replication repo
+/replicate-bug
```

Comment:
Needs newline (can we add a linter for this?)

Reply:
Yeah, we've been using one @raisedevs - I'll get on that tomorrow.